package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ipc144",
```

Comment:
None of the changes to `package.json` should be in this PR.  You can revert them:

```sh
git checkout main package.json
```

Reply:
I think now it's correct because it shows that only the file changed, right?